changelog/@unreleased/pr-1166.v2.yml

```diff
@@ -0,0 +1,6 @@
+type: fix
+fix:
+  description: Upon cancelling remoting calls, we now submit a task that closes the
```

Comment:
technically not accurate since we're not submitting anything

> Upon cancelling a remoting call via interruption, we will ensure that if the call completed successfully, its request body will be closed as it won't be processed upstream.